libcudacxx/include/cuda/std/__complex/nvbf16.h

```diff
@@ -295,6 +299,78 @@ _LIBCUDACXX_HIDE_FROM_ABI complex<__nv_bfloat16> acos(const complex<__nv_bfloat1
   return complex<__nv_bfloat16>{_CUDA_VSTD::acos(complex<float>{__x})};
 }
 
+template <>
+struct __get_complex_impl<__nv_bfloat16>
+{
+  template <size_t _Index>
```

Comment:
This should be constrained to be smaller than 2 I guess?

Reply:
It is constrained in the free `get` definition, should I check it also in `__get_complex_impl`?